app/src/main/java/ch/epfl/sweng/eventmanager/ui/eventShowcase/MapsActivity.java

```diff
@@ -0,0 +1,55 @@
+package ch.epfl.sweng.eventmanager.ui.eventShowcase;
+
+import android.support.v4.app.FragmentActivity;
+import android.os.Bundle;
+
+import com.google.android.gms.maps.CameraUpdateFactory;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.OnMapReadyCallback;
+import com.google.android.gms.maps.SupportMapFragment;
+import com.google.android.gms.maps.UiSettings;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.LatLngBounds;
+import com.google.android.gms.maps.model.MarkerOptions;
+
+import ch.epfl.sweng.eventmanager.R;
+
+public class MapsActivity extends FragmentActivity implements OnMapReadyCallback {
+
+    private GoogleMap mMap;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_maps);
+        // Obtain the SupportMapFragment and get notified when the map is ready to be used.
+        SupportMapFragment mapFragment = (SupportMapFragment) getSupportFragmentManager()
+                .findFragmentById(R.id.map);
+        mapFragment.getMapAsync(this);
+    }
+
+
+    /**
+     * Manipulates the map once available.
+     * This callback is triggered when the map is ready to be used.
+     * This is where we can add markers or lines, add listeners or move the camera. In this case,
+     * we just add a marker near Sydney, Australia.
+     * If Google Play services is not installed on the device, the user will be prompted to install
+     * it inside the SupportMapFragment. This method will only be triggered once the user has
+     * installed Google Play services and returned to the app.
+     */
+    @Override
+    public void onMapReady(GoogleMap googleMap) {
+        mMap = googleMap;
+
+        // Add a marker in Sydney and move the camera
+        LatLng epfl = new LatLng(46.518510, 6.563249);
+        mMap.addMarker(new MarkerOptions().position(epfl).title("EPFL"));
```

Comment:
Can you instead update the `Event` class to store the latitude and tag of the event location? Then you retrieve the `Event` in the activity and display the values of the event, instead of hardcoding them.

Reply:
Moreover, the comment specifies a marker in Sydney and the position is located at epfl.

Reply:
Yes, we thought about updating the Event class, but we were not sure about it. We'll do it. To answer to Antoine, we changed the marker position during our fourth commit.